mne/inverse_sparse/mxne_debiasing.py

```diff
@@ -55,7 +55,7 @@ def power_iteration_kron(A, C, max_iter=1000, tol=1e-3, random_state=0):
 
 
 @verbose
-def compute_bias(M, G, X, max_iter=1000, tol=1e-6, n_orient=1, verbose=None):
+def compute_bias(M, G, X, max_iter=10000, tol=1e-6, n_orient=1, verbose=None):
```

Comment:
This is a backward-incompatible change in the sense that it could change results right? If they are guaranteed to be better we could consider it a bugfix I suppose.